src/index.tsx

```diff
@@ -4,6 +4,6 @@ import * as ReactDOM from 'react-dom';
 
 import App from './components/App';
 
-ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
-  <App />,
-);
+ReactDOM.unstable_createRoot(
```

Comment:
unstable? :)

Reply:
@amazingalek Yeah, this is one of those experimental React things I'm playing with. It was already unstable before, just the name of the method changed.

If you're curious, some explanation:

You may notice that in `package.json` we're using an experimental version of React. In experimental versions, they don't need to mark anything as unstable, since the whole package is potentially unstable. Now, they're getting ready to move this API to the stable releases, so they need to mark it as unstable. So ironically, the `unstable_` prefix showing up now actually means the API is becoming more stable 😛

I'm using the experimental version for a reason, though. I added it back when I added logging. These experimental versions allow for asynchronous rendering, which makes it a lot easier to re-render the app very frequently (like what happens when Outer Wilds is sending logs many times a second). Without this I would have to throttle or debounce renders, which can be annoying to implement and feels a bit more sluggish.